docs/frontend-shared.md

```diff
@@ -1,67 +1,39 @@
 # Frontend Shared Package
```

Comment:
Unrelated to the changes in this PR, `docs/frontend-shared.md` is not the best file name. Something like `developing.md` would be better. I would also suggest to link to this from the README.md if that is not already the case.